app/assets/javascripts/events.coffee

```diff
@@ -1,14 +1,16 @@
 $(document).ready ->
-  $('.events').infinitescroll(
+  $container = $('.events')
+  $container.imagesLoaded ->
```

Comment:
lets add brackets before `->` in all coffee scripts in our application. IMO it's more readable doing so `() ->`